plugins/services/src/js/structs/Service.d.ts

```diff
@@ -0,0 +1,23 @@
+import Item from "#SRC/js/structs/Item";
+
+export default class Service extends Item {
+  getId(): Number;
```

Comment:
Why does this have to be `Number` instead of the primitive, `number`?

Reply:
Typescript types are still work in progress.